### PR TITLE
Fix priority between PHPDoc return type fixers

### PR DESCRIPTION
**Pull request by julienfalque**

### CODE REVIEW

File: tests/Fixtures/Integration/priority/phpdoc_to_return_type,no_superfluous_phpdoc_tags.test [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+--INPUT--
+<?php
+
+class Test {
+    /**
+     * @return int
+     */
+    public function foo() {
+    }
+}
```
Comment (SpacePossum) [inline tests/Fixtures/Integration/priority/phpdoc_to_return_type,no_superfluous_phpdoc_tags.test]: maybe best to update to
```
/**
 * @return int
 */
public function foo() {
    return 1;
}
```

so when the fixer get updates to check if some is actually returned this test still works

Comment (julienfalque) [reply tests/Fixtures/Integration/priority/phpdoc_to_return_type,no_superfluous_phpdoc_tags.test]: Updated.

**Comment by SpacePossum** [comment]
Thanks @julienfalque.

**Comment by SpacePossum** [comment]
thanks @keradus as well :)

**Pull request merged**